app/models/like.rb

```diff
@@ -0,0 +1,11 @@
+class Like < ApplicationRecord
+  belongs_to :users
```

Comment:
- [ ] the like belongs to User through the author, kindly make sure to implement this so the association setup correctly 

```ruby
belongs_to :author, class_name: 'User'
```

Reply:
well resolved! ✔️